lib/msal-browser/src/utils/BrowserConstants.ts

```diff
@@ -106,7 +106,8 @@ export enum ApiId {
 export enum InteractionType {
     Redirect = "redirect",
     Popup = "popup",
-    Silent = "silent"
+    Silent = "silent",
+    None = "none"
```

Comment:
Did you get a chance to see if this could be added in the 1p lib instead of here?

Reply:
Yes, you can't extend enums and it wouldn't make sense to add as a separate enum.